### PR TITLE
Dcs 737 date and time available page

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: backend/services/bookingService.ts [hunk 1]
```diff
@@ -44,7 +44,14 @@ export = class BookingService {
       const location = locations.find(loc => loc.locationId === appointment.locationId)
       const prisonRoom = location?.userDescription || location?.description || ''
       const description = `${prisonRoom} - ${Time(appointment.startTime)} to ${Time(appointment.endTime)}`
-      return { prisonRoom, description, startTime: Time(appointment.startTime), endTime: Time(appointment.endTime) }
+      const timings = `${Time(appointment.startTime)} to ${Time(appointment.endTime)}`
```
Comment (andrewrlee) [inline backend/services/bookingService.ts:47]: could use this new `timings` variable in the `description` literal?

Comment (louisenorris) [reply backend/services/bookingService.ts:47]: Ah instead of starttime and endtime?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: backend/services/bookingService.ts (hunk 1 above)
Comment (andrewrlee) [reply backend/services/bookingService.ts:47]: yeah.. `const description = `${prisonRoom} - ${timings}`

**Pull request merged**